src/main/java/org/broadinstitute/dsm/db/dto/kit/ClinicalKitDto.java

```diff
@@ -46,23 +39,40 @@
     @SerializedName("accession_number")
     String accessionNumber;
 
-    String instanceName;
+    @SerializedName("collection_date")
+    String collectionDate;
+
+    @SerializedName("kit_label")
+    String mfBarcode;
 
     public ClinicalKitDto(){}
 
     public void setSampleType(String kitType){
         switch (kitType.toLowerCase()){
             case "saliva":
-                this.sampleType = NORMAL;
+                this.sampleType = "Normal";
```

Comment:
I didn't understand why we changed them... ( not that the change. breaks anything, just didn't understand the reason behind it ☺️ )